cpu.go

```diff
@@ -0,0 +1,22 @@
+//go:build !js
+// +build !js
+
+package qtls
+
+import (
+	"golang.org/x/sys/cpu"
+)
+
+var (
+	CPUHasAES       = cpu.X86.HasAES
+	CPUHasAESCBC    = cpu.S390X.HasAESCBC
+	CPUHasAESCTR    = cpu.S390X.HasAESCTR
+	CPUHasAESGCM    = cpu.S390X.HasAESGCM
+	CPUHasPCLMULQDQ = cpu.X86.HasPCLMULQDQ
+	CPUHasPMULL     = cpu.ARM64.HasPMULL
```

Comment:
Why is this using `cpu.ARM64`?

Reply:
@egonelbre The code is the same as was there before - just moved to a different location.

https://github.com/marten-seemann/qtls-go1-17/pull/2/files#diff-aa99d9b5e777a57d11244c217daa11772dae47d3f935d51f06c20fb0557083f9L370